versions/3_12/samples/ecom/get_payment_details.py

```diff
@@ -0,0 +1,38 @@
+from datetime import datetime
+import grpc
+
+import kody_clientsdk_python.ecom.v1.ecom_pb2 as kody_model
+import kody_clientsdk_python.ecom.v1.ecom_pb2_grpc as kody_client
+from kody_clientsdk_python.sdk.common.pagination_pb2 import PageCursor
+
+def get_payment_details():
+    # TODO: Replace this with the testing or live environment
+    address = "grpc-staging.kodypay.com"
```

Comment:
Update the README as well? Looks not correct with your new updates
https://github.com/KodyPay/kody-clientsdk-python/tree/main/versions/3_12/samples#readme